features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/js/edit.asset.js

```diff
@@ -185,35 +187,39 @@ $(function() {
     });
     
     function addToClaimsTable(claim,clickable){
-    	var propertyCount = $('#claimPropertyCounter');
-
-	var i = propertyCount.val(); 
-	var currentCount = parseInt(i);
-
-	currentCount = currentCount + 1;
-	propertyCount.val(currentCount);
-
-	$('#claimTableId').hide();
-	if(clickable){
-		$('#claimTableTbody').append($('<tr id="claimRow' + i +'" class="claimRow">'+
-	    		'<td style="padding-left: 40px ! important; color: rgb(119, 119, 119); font-style: italic;">'+
-	    		claim + '<input type="hidden" name="claimPropertyName' + i + '" id="claimPropertyName' + i + '"  value="' + claim + '"/> '+
-	    		'</td>'+
-	    		'<td>'+
-	    		'<a href="#"  onclick="removeClaim(' + i + ');return false;"><i class="icon-remove-sign"></i>  Delete</a>' +
-	    		'</td>'+
-	    		'</tr>'));
-	}else{
-		$('#claimTableTbody').append($('<tr id="claimRow' + i +'" class="claimRow">'+
-	    		'<td style="padding-left: 40px ! important; color: rgb(119, 119, 119); font-style: italic;">'+
-	    		claim + '<input type="hidden" name="claimPropertyName' + i + '" id="claimPropertyName' + i + '"  value="' + claim + '"/> '+
-	    		'</td>'+
-	    		'<td>'+
-	    		'<a href="#" style="pointer-events: none; cursor: default;color:#C4C4C4"  onclick="removeClaim(' + i + ');return false;"><i class="icon-remove-sign"></i>  Delete</a>' +
-	    		'</td>'+
-	    		'</tr>'));
-	}
-	$('#claimTableTbody').parent().show();
+        var isAlreadyExist = $.inArray(claim, addedClaimList);
```

Comment:
Given that there is a language library, It is wise to avoid third party utils whenever possible.

jQuery.inArray() shouldd be able to be replaced with Array.indexOf() [1]

[1] - https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf